test/test_base.go

```diff
@@ -39,7 +39,8 @@ var (
 		false,        // IBMz
 		false,        // PPC
 		false,        // Openshift
-		1}            // TimeoutMultiplier
+		1,            // TimeoutMultiplier
+		false}        // GlobalOperator
```

Comment:
Do you think it's better to store the `}` on a single line, so, you'll just add new values and do not keep the previous line as changed?

Reply:
Discussed in google chat.